src/engine/math_util.h

```diff
@@ -551,9 +551,9 @@ void mtxf_mul(Mat4 dest, Mat4 a, Mat4 b);
 void mtxf_scale_vec3f(Mat4 dest, Mat4 mtx, Vec3f s);
 void mtxf_mul_vec3s(Mat4 mtx, Vec3s b);
 
-extern void mtxf_to_mtx_asm(register void *dest, register void *src);
+extern void mtxf_to_mtx_fast(register s16 *dest, register float *src);
 ALWAYS_INLINE void mtxf_to_mtx(register void *dest, register void *src) {
-    mtxf_to_mtx_asm(dest, src);
+    mtxf_to_mtx_fast((s16*)dest, (float*)src);
```

Comment:
this still better always inline?

Reply:
It's just inlining the function call to another function call, no different than externing the function and calling it directly (casts between types of pointers are free operations). I just did it this way to keep it the way the repo currently worked.

Reply:
word sounds good!